web/src/copy_and_paste.js

```diff
@@ -341,7 +346,30 @@ export function paste_handler(event) {
     }
 
     if (clipboardData.getData) {
+        const $textarea = $(event.currentTarget);
+        const paste_text = clipboardData.getData("text");
+        // Trim the paste_text to handle sloppy copying
+        const trimmed_paste_text = paste_text.trim();
         const paste_html = clipboardData.getData("text/html");
+
+        // Match for anything that resembles a URI, something://something
+        const uri_regex = /^\w+:\/\/\S+$/;
```

Comment:
```suggestion
        // Match for anything that resembles a URL, something://something
        const url_regex = /^\w+:\/\/\S+$/;
```

The term "URI" has thankfully been made obsolete, and "URL" is the standard term. See #23380 and discussion at https://chat.zulip.org/#narrow/stream/92-learning/topic/URLs/near/794826 .

Reply:
Oof. Yeah, that's old data rattling around in my head. I'll gladly push a fix for this.

Reply:
Yeah, I was quite happy when I discovered a couple of years ago (in that thread) that I officially could stop caring about the concept of a "URI" :slightly_smiling_face:

Reply:
Agreed. "URI" always hit my ears as an unpleasant medical term. But the W3C was so adamant about the change that several synapses in my brain have been permanently altered.